src/renderer/delta-remote.ts

```diff
@@ -259,6 +259,18 @@ class DeltaRemote {
   call(fnName: string, ...args: any[]): Promise<any> {
     return _callDcMethodAsync(fnName, ...[...arguments].slice(1))
   }
+
+  on(channel: string, listener: (event: Electron.IpcRendererEvent, ...args: any[]) => void) {
+    ipcBackend.on(channel, listener)
+  }
+
+  once(channel: string, listener: (event: Electron.IpcRendererEvent, ...args: any[]) => void) {
```

Comment:
```suggestion
  once(...args: Parameters<typeof ipcRenderer.once>) {
```
you can simplify this.